docs/components/rpa/getting-started.md

```diff
@@ -8,21 +8,24 @@ description: "Learn how to create your first script, test your script, and autom
 New to RPA? Visit the [overview page](/components/rpa/overview.md) to get familiar with Robotic Process Automation (RPA) capabilities.
 :::
 
-### Create your first script
+The RPA worker is available on all major platforms (Windows, Linux, and Mac). This allows you to automate your applications on their native platforms. In most cases, this will be Windows. For console applications or browser automation, you can use a more light-weight distribution, such as the [Docker image](/self-managed/setup/deploy/other/docker.md).
+
+## Create your first script
 
 <!-- TODO(marstamm): The modeler flag is only required in the alpha releases. Remove this caution for 8.7 minor relase -->
 
 :::caution
 In Camunda Modeler 5.33.1, the RPA editor is enabled only if the `--no-disable-rpa` flag is set. Visit the [Modeler documentation](/components/modeler/desktop-modeler/flags/flags.md) to learn about setting flags.
 :::
+
 To get started with RPA, you first need to write an RPA script. [Camunda Modeler](/components/modeler/about-modeler.md) offers an interface for editing and testing your scripts:
 
 1. **Download Camunda Modeler**: Download the latest version of Camunda Modeler from the [Camunda website](https://camunda.com/download/modeler/). As RPA scripts are run locally on your machine, the RPA editor is only available in [Desktop Modeler](/components/modeler/desktop-modeler/index.md).
 2. **Open the RPA script editor**: Open Desktop Modeler and navigate to the RPA script editor under **Testing**.
    ![A screenshot of the RPA Editor in the Camunda Modeler](img/rpa-editor-initial.png)
 3. **Start writing your RPA script using Robot Framework**: Use the interface provided to write your first RPA script. Scripts use the [Robot Framework](https://robotframework.org/) syntax.
 
-### Test your script
+## Test your script
 
 Once you have written your script, you can test it on a local RPA worker.
 
```

Comment:
Below we say "4. Start the worker by running the executable." -- what do we mean by this? Is this a particular line everyone uses in the terminal or is this unique to every user?

Reply:
This has changed since we wrote it 🙈 

I adjusted it: 162d89e66ab72e49cc2d315f3dd89ec2ad9b7e46

Reply:
I adjusted the wording to make it clearer what to execute. The filename and how to start it depends on your operating system

d7991a21bf577e1f8562d6dfc2e45df20f603af0